src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java

```diff
@@ -62,14 +65,28 @@ public static SelenideElement getDataTypeWithName(String datatypeName) {
                 .$$(By.className("api-definition")).filter(text(datatypeName)).first();
     }
 
-    public static void createDataType(String name, String description, Boolean isRest) {
+    public static void createDataType(String name, String description, String example, Boolean isRest) {
         log.info("Creating data type with name {} and description {} and rest resources are {}", name, description, isRest);
 
-        CommonUtils.getLabelWithName("name", CommonUtils.getAppRoot()).setValue(name);
+        SelenideElement entityForm = CommonUtils.getAppRoot().$(MainPageElements.ENTITY_EDITOR_FORM);
+
+        CommonUtils.getLabelWithName("name", entityForm).setValue(name);
+
+        if (!description.isEmpty()) {
```

Comment:
If description/example parameters are optional, would it make sense to overload the method for easier use? These ifs don't look good. I understand the usage, but isn't there a better way how to do it?

Reply:
Added docs as we discussed